index.js

```diff
@@ -2,12 +2,7 @@
 
 const yargs = require('yargs/yargs');
 const flatten = require('flat');
-const castArray = require('lodash/castArray');
-const some = require('lodash/some');
-const isPlainObject = require('lodash/isPlainObject');
-const camelCase = require('lodash/camelCase');
-const kebabCase = require('lodash/kebabCase');
-const omitBy = require('lodash/omitBy');
+const {castArray, some, isPlainObject, camelCase, kebabCase, omitBy} = require('lodash/lodash.min');
```

Comment:
this will result in much more javascript being parsed. lodash splits out the methods for this purpose

Reply:
I found that the extra filesystem calls, not JS parsing, was the performance killer.

Reply:
@fringd here's a quick test you can try yourself.

```
time node -e "for(const lib of process.argv.slice(2)) require('lodash/' + lib)" castArray some isPlainObject camelCase kebabCase omitBy ; time node -e "for(const lib of process.argv.slice(2)) require('lodash/' + lib)" lodash.min.js
```